comps/dataprep/redis/langchain/prepare_doc_redis.py

```diff
@@ -187,5 +211,21 @@ def process_files_wrapper(files):
     raise HTTPException(status_code=400, detail="Must provide either a file or a string list.")
 
 
+@register_microservice(
```

Comment:
Are we implementing a GET request using a POST endpoint?

Reply:
Also one more concern - One microservice runs the 2 uvicorn server on 2 different ports.

Reply:
Shall we prioritize adding of the GET/UPDATE/DELETE method in current micro-service design. This will be a cleaner design.